enterprise-cfengine-guide/federated-reporting.markdown

```diff
@@ -23,13 +23,33 @@ configure and connect the Superhub and Feeder hubs. For Feeder hubs with an
 earlier version than 3.14.0 some manual steps must be taken. Links to these
 are provided at each stage of installation and setup that follows.
 
+* [Requirements][Federated Reporting#Requirements]
 * [Installation][Federated Reporting#Installation]
 * [Setup][Federated Reporting#Setup]
 * [Operation][Federated Reporting#Operation]
 * [API Setup][Federated Reporting#API Setup]
 * [Disable Feeder][Federated Reporting#Disable Feeder]
 * [Uninstall][Federated Reporting#Uninstall]
 
+## Requirements ##
+
+If your hub will have SELinux enabled you must install the `semanage` command.
```

Comment:
I would change the wording slightly.

If your hub will have SELinux enabled, the `semanage` command must be installed.